### ISSUE TITLE
[link-rel-noopener] Add whitelist support

### ISSUE THREAD

**Issue by nag5000** [ember-template-lint/ember-template-lint#1478]
In my app, in most cases, external links lead to my own subdomains or trusted domains for which `rel="noopener noreferrer` is unnecessary.

It would be great to be able to turn off this rule for trusted domains.

**Comment by rwjblue** [comment]
Ya, seems reasonable to me. So the work here would be to add a new configuration option for `trustedDomains` and if the `href` was for one of those trusted domains the rule would not flag an error.

**Comment by nag5000** [comment]
Yes, something like that.., but I'm afraid there is one case that will not be covered by this implementation. 

It's the case when `href` is bound to a variable or another handlebars expression: 
`href={{url}}`, `href={{t "some.translation.link"}}` or `href={{concat "example.com?q=" query}}` 
(when I'm sure that the result of the expression is a trusted URL). 

Anyway, I don't see any good non-rocket-science solution for this situation, except simply to use control comments `{{!-- template-lint-disable ... --}}` or to disable this rule globally at all.
